src/components/banner/banner.scss

```diff
@@ -82,6 +82,7 @@
     margin: 0 auto;
     width: 100%;
     z-index: 1;
+    pointer-events: none;
```

Comment:
This removes the hover on the buttons attached to the banner.   I think just a `z-index` adjustment will fix this issue instead:
```
.media--video .video-controls {
    z-index: 2;
}
```